python/extensions.bzl

```diff
@@ -127,3 +152,48 @@ pip = module_extension(
         "parse": tag_class(attrs = _pip_parse_ext_attrs()),
     },
 )
+
+def _host_hub_impl(repo_ctx):
+    repo_ctx.file("BUILD.bazel", """\
+package(default_visibility = ["//visibility:public"])
+exports_files(["python"])""")
+
+    # use the same functions that we have to
+    # create the toolchains with, that create
+    # the Label "interpreter".
+    (os, arch) = get_host_os_arch(repo_ctx)
+    host_platform = get_host_platform(os, arch)
+
+    interpreters = []
+    for config_label in repo_ctx.attr.interpreters:
+        if host_platform in str(config_label):
+            interpreters.append(config_label)
+
+    num = len(interpreters)
+    if num == 1:
+        # We found the interpreter, so we create the symlink
+        # Regardless of the OS, we need a symlink named "python"
+        # as we have code in the pip rules that is hardcoded on the
+        # name.
+        repo_ctx.symlink(interpreters[0], "python")
+    elif num == 0:
+        fail("Unable to find interpreter, that matches the host os and architecture, in the argument interpreters")
```

Comment:
Please include the ones that were considered in the error message. "None of these were deemed compatible: <list of the labels>".